src/sentry/static/sentry/app/views/projectInstall/newProject.jsx

```diff
@@ -13,9 +13,9 @@ const NewProject = ({organization}) => (
       <Content>
         <DocumentTitle title="Sentry" />
         <CreateProject
-          hasIssueAlertOptionsEnabled={organization.features.includes(
```

Comment:
It's a good thing we refactored this logic to be a prop

Reply:
Good thing my mentor suggested the refactoring